src/ledger/ledger.css

```
@@ -0,0 +1,58 @@
+.ledger__controls button { padding: 6px 12px; cursor: pointer; }
+.ledger__controls button[aria-pressed="true"] { font-weight: 600; }
+.ledger__controls button:disabled { cursor: default; opacity: 0.5; }
+
+.ledger__rows { list-style: none; margin: 0; padding: 0; }
+.ledger__row {
+  display: grid;
+  grid-template-columns: 8em 1fr auto;
+  gap: 4px 12px;
+  align-items: baseline;
+  padding: 8px 4px;
+  border-top: 1px solid color-mix(in srgb, currentColor 15%, transparent);
```

Stylelint (17.14.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)